rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/evaluation/SmeupInterpreterTest.kt

```diff
@@ -219,4 +219,14 @@ open class SmeupInterpreterTest : AbstractTest() {
             "smeup/T10_A60_P04F".outputOf()
         }
     }
+
+    @Test
+    fun executeT10_A20_P19() {
+        val expected = listOf("1020")
+        assertEquals(expected, "smeup/T10_A20_P19".outputOf())
+
+        assertFailsWith<Exception> {
```

Comment:
Remove this assert. It is unhelpful because this check was already made also before your changes, in addition, `T10_A20_P19F` could cause a nameclash with test unit written in as400.